L2-Zad.5-EvenAndOddNumbers/src/main/java/Main/EvenOrOddCheck2.java

```diff
@@ -0,0 +1,10 @@
+package Main;
+
+public class EvenOrOddCheck2 {
```

Comment:
to jest o wieele ładniejsze rozwiązanie niżto "EvenOrOddCheck" gdzie zwracasz string.

Są dwie opcje do zwrócenia - albo liczba jest parzysta, albo nie. Typ boolean idealnie tutaj pasuje. :) 

Usuń proszę tę wersję ze stringiem.